### PR TITLE
Enable PWA Support

### PR DESCRIPTION
**Pull request by 0916dhkim**
### Changes
- Add `public/service-worker.js`.
- Register service worker in `src/index.tsx`.

### Notes
- Web app manifest is already supported in CRA.
- Cannot use typescript inside service worker.

**This PR is made on top of https://github.com/dooboolab/dooboo-native-ts/pull/162**

### CODE REVIEW

**Comment by 0916dhkim** [comment]
@hyochan Please review this PR. Note that I made this branch on top of your branch (not master).

**Comment by hyochan** [comment]
> @hyochan Please review this PR. Note that I made this branch on top of your branch (not master).

This is the way to go! You did great

File: .eslintrc.js [hunk 1]
```diff
@@ -5,4 +5,12 @@ module.exports = {
...
     'eslint-comments/no-unused-disable': 0,
   },
+  overrides: [
+    {
+      files: ['*.js'],
+      rules: {
+        '@typescript-eslint/explicit-function-return-type': 0,
+      },
+    },
+  ],
```
Comment (hyochan) [inline .eslintrc.js:15]: Lets handle this in https://github.com/dooboolab/eslint-config/commit/3eb4f36511da28e0f3733cef3ea1bf8b3fbed572

**Review by hyochan — APPROVED** [review]

File: .eslintrc.js (hunk 1 above)
Comment (hyochan) [reply .eslintrc.js:15]: I'll handle this!

**Pull request merged**